activesupport/lib/active_support/core_ext/securerandom.rb

```diff
@@ -16,7 +16,7 @@ module SecureRandom
   #
   #   p SecureRandom.base58 # => "4kUgL2pdQMSCQtjE"
   #   p SecureRandom.base58(24) # => "77TMHrHJFvFDwodq8w7Ev2m7"
-  if RUBY_VERSION >= "3.3"
+  if SecureRandom::VERSION >= "0.3"
```

Comment:
I'm worried that this will lead to bug reports when securerandom 0.10 is released, is that ok?

Reply:
Yeah, it's very unlikely to happen any time soon. But to be frank with you I'm tempted to just add secure random as a dependency to require `>= 0.3`.